src/7E9D0.c

```diff
@@ -191,10 +398,14 @@ void set_action_state(s32 actionState) {
 
 void update_locomotion_state(void) {
     PlayerStatus* playerStatus = &gPlayerStatus;
-    do { } while (0); // required to match
 
-    set_action_state((!is_ability_active(ABILITY_SLOW_GO)
-                      && (SQ(playerStatus->stickAxis[0]) + SQ(playerStatus->stickAxis[1]) >= 0xBD2)) ? ACTION_STATE_RUN : ACTION_STATE_WALK);
+    if (!is_ability_active(ABILITY_SLOW_GO) &&
+        SQ(playerStatus->stickAxis[0]) + SQ(playerStatus->stickAxis[1]) >= 0xBD2)
```

Comment:
```suggestion
        SQ(playerStatus->stickAxis[0]) + SQ(playerStatus->stickAxis[1]) >= 3026)
```
Surprised the comparison value 3026 isn't a square number

Reply:
its 55**2 + 1